plugins/woocommerce/tests/php/src/Internal/Admin/ReviewsTest.php

```diff
@@ -16,6 +16,26 @@
  */
 class ReviewsTest extends WC_Unit_Test_Case {
 
+	/**
+	 * Sets the global vars before each test.
+	 */
+	public function setUp() : void {
+		global $pagenow;
+
+		$this->old_pagenow = $pagenow;
+
+		parent::setUp();
+	}
+
+	/**
+	 * Restores the global vars after each test.
+	 */
+	public function tearDown() : void {
+		global $pagenow;
+
+		$pagenow = $this->old_pagenow; // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
Just double-checking, shouldn't we call the parent class `tearDown()` method?

Reply:
Done in https://github.com/godaddy-wordpress/woocommerce/pull/34/commits/f0c8f6976fba3be6254b313dda1974113767c38e.